internal/gitrepo/gitrepo.go

```diff
@@ -373,7 +389,10 @@ func (r *Repository) GetCommitsForReleaseID(releaseID string) ([]object.Commit,
 		}
 
 		if gotReleaseID {
-			commits = append(commits, *candidateCommit)
+			commits = append(commits, Commit{
```

Comment:
```commits = append(commits, &Commit{...})```

re: pointers comment above

Reply:
It looks like this is still `commits = append(commits, Commit{` and needs to be changed.

Reply:
Sorry I missed this, I believe this time it changed.